src/index.html

```diff
@@ -31,10 +31,36 @@
       body:not(:has(app-root router-outlet ~ *)) > .loading {
         display: flex;
       }
+
+      .loading::after {
+        content: '...';
+        display: inline-block;
+        width: 24px;
+        text-align: left;
+      }
+
+      .loading::after {
+        animation: loadingDots 1.5s infinite;
+      }
+
+      @keyframes loadingDots {
+        0%, 20% {
+          content: '.';
+        }
+        40% {
+          content: '..';
+        }
+        60% {
+          content: '...';
+        }
+        80%, 100% {
+          content: '.';
+        }
+      }
```

Comment:
We have two loading components in sparkle we can utilize

`<spk-progress-bar class="indeterminate primary" />`

Or 

`<spk-spinner class="primary"></spk-spinner>`

Mostly so we're consistent with what UI we use